src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -88,14 +88,237 @@
+                {{ transformedResult.up }}
+            <span class="fw-bold text-success">
+              up</span>
+          </span>
+          <span *ngIf="transformedResult.in !== transformedResult.up"
+                class="ms-2">
+                {{ transformedResult.in }}
+            <span class="fw-bold text-success">
+              in</span></span>
+          <span *ngIf="transformedResult.down"
+                class="ms-2">
+                {{ transformedResult.down }}
+            <span class="fw-bold text-danger me-2">
+              down</span></span>
+          <span *ngIf="transformedResult.out"
+                class="ms-2">
+                {{ transformedResult.out }}
+            <span class="fw-bold text-danger me-2">
+              out</span>
+          </span>
+          <span *ngIf="transformedResult.nearfull"
+                class="ms-2">
+                {{ transformedResult.nearfull }}
+            <span class="fw-bold text-warning me-2">
+              nearfull</span></span>
+          <span *ngIf="transformedResult.full"
+                class="ms-2">
+                {{ transformedResult.full }}
+            <span class="fw-bold text-danger">
+              full
+            </span>
+          </span>
+        </div>
+      </li>
+      <!-- Pools -->
+      <li *ngIf="healthData.pools"
+          class="border-top list-group-item">
+        <div class="inventory-item d-flex pl-1"
+             id="pools-item">
+          <div class="ms-2 me-auto"
+               i18n>
+            <a routerLink="/pool">
+            {{ healthData.pools.length }} Pools</a>
+          </div>
+          <span>
+            {{ healthData.pools.length }}
+            <i class="text-success"
+               *ngIf="healthData.pools.length > 0"
+               [ngClass]="[icons.success]"></i>
+          </span>
+        </div>
+      </li>
+      <!-- PG Info -->
+      <li *ngIf="totalPgs > 0"
+          class="border-top list-group-item">
+        <div class="inventory-item d-flex pl-1"
+             id="pg-item">
+          <div class="ms-2 me-auto"
+               i18n>
+          {{ totalPgs | dimless }} PGs
+          </div>
+          <span *ngIf="totalPgs === 0">
+            {{ totalPgs }}
+          </span>
+          <span *ngFor="let pgStatesText of categoryPgAmount | keyvalue">
+            {{ pgStatesText.value | dimless }}
+            <i class="text-success"
+               [ngClass]="[icons.success]"
+               *ngIf="pgStatesText.key === 'clean'"></i>
+            <i class="text-warning"
+               [ngClass]="[icons.warning]"
+               *ngIf="pgStatesText.key === 'warning'"></i>
+            <i class="text-danger"
+               [ngClass]="[icons.danger]"
+               *ngIf="pgStatesText.key === 'unknown'"></i>
+            <i class="text-warning"
+               [ngClass]="[icons.spinner, icons.spin]"
+               *ngIf="pgStatesText.key === 'working'"></i>
+          </span>
```

Comment:
I think that the flickering icon might be related to this ngFor. Couple of ideas:
* Using ngFor's [trackBy](https://blog.bitsrc.io/angular-optimization-use-trackby-option-for-ngfor-directive-72c9509b2be9) to help change detector identify real changes.
* Create a service/pipe/whatever that wraps the PGStatesText and delivers something ready to use in an ngIf:
```html
<cdInventoryRow *ngIf="pgInfo$ | async as pgInfo"
    i18n-item="PGs"
    [count]="pgInfo | cdPgInfoToCount"
></cdInventoryRow>
```

Reply:
Used a pipe. Now the flickering is gone.